content/articles/99999980-upgrade-python-on-linux.md

```diff
@@ -156,11 +156,19 @@ sudo apt install python3.10-venv
 
 All should be done now. It is complicated, but this is how you update Python to latest version.
 
+### Fix `x86_64-linux-gnu-gcc`... error
+When you try to install specific dependencies they will look for an updated C dependency used for compilation. This must be manually installed by running
+```bash
+sudo apt install build-essential
+sudo apt install python3.10-dev
+```
+
 ### Extra
 If you have [oh-my-zsh](https://ohmyz.sh/) installed, you can avoid typing out `python3` by running
 ```bash
 echo "alias py=/usr/bin/python3" >> ~/.zshrc
 echo "alias python=/usr/bin/python3" >> ~/.zshrc
 ```
-Now you can run your files with `py` or `python`.
-To help you skip the hassle in virtual environment setup and package management, consider using [PDM](https://pdm.fming.dev/latest/).
+Now you can run your files with `py` or `python`. To help you skip the hassle in virtual environment setup and package management, consider using [PDM](https://pdm.fming.dev/latest/). 
+
+You can also learn how to [set up a WSL terminal](https://github.com/hdadhich01/wsl-terminal) to have a full-fledged Python development environment.
```

Comment:
I don't think this is relevant and specific enough to the article scope and should not be included.